src/components/NurseryWithdrawals/WithdrawalLogRenderer.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@mui/styles';
+import { APP_PATHS } from 'src/constants';
+import CellRenderer, { TableRowType } from '../common/table/TableCellRenderer';
+import { RendererProps } from '../common/table/types';
+import { Theme } from '@mui/material';
+import { Button } from '@terraware/web-components';
```

Comment:
as far as import patterns.. we also re-export this from `src/components/common/button/Button.tsx`. Not sure which is our preferred pattern or the history behind those re-exports. Personally, I like this direct import from `web-components` better.

Reply:
Yes, I think we should always import from '@terraware/web-components'; It looks cleaner